### PR TITLE
tweaking display of server and connections

### PR DESCRIPTION
**Pull request by NamespaceV**

### CODE REVIEW

File: Scripts/Multiplayer/server.gd [hunk 1]
```diff
@@ -20,19 +22,31 @@ func _init():
 	LobbySetUnitCommand.register(incoming_commands)
 	ClientRequestedMoveCommand.register(incoming_commands)
 
+func _ready():
+	var request = HTTPRequest.new()
+	add_child(request)
+	request.request_completed.connect(self.onRequestCompleted)
+	request.request("https://api.ipify.org")
```
Comment (PieroganoReggiano) [inline Scripts/Multiplayer/server.gd:29]: what is this, why

Comment (NamespaceV) [reply Scripts/Multiplayer/server.gd:29]: discussed on voice
will remove this call from here and add a button that makes a call
with some more readable explanation and more config for the endpoint

Comment (NamespaceV) [reply Scripts/Multiplayer/server.gd]: added fixes

Comment (NamespaceV) [reply Scripts/Multiplayer/server.gd]: Btw the service i used is open source, checks a header in http and writes it back:
https://github.com/rdegges/ipify-api/blob/master/api/get_ip.go#L33C1-L34C1

**Pull request merged**